recipe/patches/0005-Disable-making-non-core-entry-scripts.patch

```diff
@@ -1,23 +1,25 @@
-From c7dbc5b7e1f6e76be9ce245735589141ff415077 Mon Sep 17 00:00:00 2001
+From 51552e702ce81936906030bde40f5e2ab2660b7e Mon Sep 17 00:00:00 2001
 From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
 Date: Tue, 10 Nov 2020 23:26:35 +0300
 Subject: [PATCH 05/13] Disable making non-core entry scripts
 
 Signed-off-by: Vasily Litvinov <vasilij.n.litvinov@intel.com>
 ---
- python/setup.py | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
+ python/setup.py | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
 
 diff --git a/python/setup.py b/python/setup.py
-index d64d8402e..b0a20062a 100644
+index 2c82ad1d9..63ab0d09a 100644
 --- a/python/setup.py
 +++ b/python/setup.py
-@@ -462,7 +462,7 @@ setuptools.setup(
+@@ -450,8 +450,8 @@ setuptools.setup(
      entry_points={
          "console_scripts": [
              "ray=ray.scripts.scripts:main",
--            "rllib=ray.rllib.scripts:cli [rllib]", "tune=ray.tune.scripts:cli"
-+#            "rllib=ray.rllib.scripts:cli [rllib]", "tune=ray.tune.scripts:cli"
+-            "rllib=ray.rllib.scripts:cli [rllib]", "tune=ray.tune.scripts:cli",
+-            "ray-operator=ray.operator:main"
++#            "rllib=ray.rllib.scripts:cli [rllib]", "tune=ray.tune.scripts:cli",
++#            "ray-operator=ray.operator:main"
```

Comment:
@vnlitvinov 
I'm not sure what your thought was regarding "Disable making non-core entry scripts". Not sure we can even support `ray-operator` right away, It does not seem to have a separate package (e.g. `pip install ray[operator]`), but [starts off](https://github.com/ray-project/ray/blob/ray-1.1.0/python/ray/operator.py#L24) `import kubernetes` and also looks like it depends on `ray-autoscaler`.

Reply:
I've somehow missed this discussion, sorry about that.

My idea of splitting non-core entry scripts was to pull each of them into a separate meta-package which would bring all the dependencies with itself.